client/src/components/admin/tables/ProjectsTable.tsx

```diff
@@ -52,6 +55,19 @@ const ProjectsTable = () => {
         fetchProjects();
     }, [fetchProjects]);
 
+    // Fetch flags on load
+    useEffect(() => {
+        async function getFlags() {
+            const res = await getRequest<Flag[]>('/admin/flags', 'admin');
+            if (res.status !== 200) {
+                errorAlert(res);
+            }
+            setFlags(res.data as Flag[]);
+        }
+
+        getFlags();
+    }, []);
+    
```

Comment:
honestly this should be in the [zustand global store](https://github.com/pmndrs/zustand) instead of passing it into every single project row...

It's fine how you have it tho; just resolve this comment but if you could make an issue for it that would be nice

Reply:
The store is at [`store.tsx`](https://github.com/hackutd/jury/blob/53c28b6c4d7291ef3f3e8fa3f1210506ee240d3d/client/src/store.tsx)